### PR TITLE
[Refactoring] Added/Revised Comments for the exported functions (#353)

### PR DESCRIPTION
**Pull request by nitu-s-gupta**
Signed-off-by: Nitu Gupta <nitu.gupta@samsung.com>

# Description
To remove Lint warnings ,comments have been added/revised
    1. internal/common/resourceutil/cpu/
    2. internal/common/types/configuremgrtypes/
    3. internal/controller/discoverymgr/
    4. internal/controller/scoringmgr/
    5. internal/controller/servicemgr/
    6. internal/orchestrationapi/
    7. internal/restinterface/externalhandler/
    8. internal/restinterface/internalhandler/
    9. internal/restinterface/resthelper/
    10. internal/restinterface/route/route.go

Fixes # (#353)

## Type of change
- [X] Bug fix (non-breaking change which fixes an issue)
- [X] Code cleanup/refactoring

# How Has This Been Tested?
$ golint {Folderpath to which lint check has to be applied}

**Test Configuration**:
* Firmware version: (Ubuntu 20.04, etc.)
* Hardware: (x86-64)
* Edge Orchestration Release: (v1.0.0)

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -9,7 +9,6 @@ require (
 	github.com/Azure/go-ansiterm v0.0.0-20210617225240-d185dfc1b5a1 // indirect
 	github.com/Microsoft/go-winio v0.4.16 // indirect
 	github.com/StackExchange/wmi v0.0.0-20190523213315-cbe66965904d // indirect
-	github.com/axw/gocov v1.0.0 // indirect
```
Comment (t25kim) [inline go.mod:12]: Please do not exclude gocov and gocov-html from go.mod.

**Review by t25kim — COMMENTED** [review]
Thanks for the quick update!
Please take a look at my comment.

File: go.mod (hunk 1 above)
Comment (nitu-s-gupta) [reply go.mod]: Done

**Review by t25kim — APPROVED** [review]
LGTM Thanks!

**Review by tdrozdovsky — APPROVED** [review]
Thank you!

File: internal/common/resourceutil/cpu/cpu.go [hunk 2]
```diff
@@ -10,24 +10,38 @@ import (
...
 	GESTNICE
 )
 
 const (
-	CPU_IDLE = iota
-	CPU_NONIDLE
-	CPU_TOTAL
+	// CPUIdle is the key idle cpu. value set to 0
+	CPUIdle = iota
+	//CPUNonidle is key for non idle cpu
```
Comment (MoonkiHong) [inline internal/common/resourceutil/cpu/cpu.go:38]:
```suggestion
	// CPUNonidle is key for non idle cpu
```

@nitu-s-gupta One last checker, although this is a trivial change. 😄

File: internal/controller/servicemgr/types.go [hunk 3]
```diff
@@ -41,6 +41,7 @@ const (
 	// ConstKeyUserArgs is key of user argument
 	ConstKeyUserArgs = "UserArgs"
 
+	//ConstKeyRequester is key for requester
```
Comment (MoonkiHong) [inline internal/controller/servicemgr/types.go:44]:
```suggestion
	// ConstKeyRequester is key for requester
```

@nitu-s-gupta Same trivial suggestion from me. 😄

File: internal/restinterface/resthelper/client/httphelper/httphelper.go [hunk 4]
```diff
@@ -25,7 +25,8 @@ import (
 
 var client *http.Client
 
-type HttpHelper struct{}
+//HTTPHelper struct
```
Comment (MoonkiHong) [inline internal/restinterface/resthelper/client/httphelper/httphelper.go:28]:
```suggestion
// HTTPHelper struct
```

@nitu-s-gupta Same trivial suggestion from me. 😄

File: internal/restinterface/resthelper/client/tlshelper/tlshelper.go [hunk 5]
```diff
@@ -35,6 +35,7 @@ var (
 	wellKnownPort map[string]string
 )
 
+//TLSHelper struct
```
Comment (MoonkiHong) [inline internal/restinterface/resthelper/client/tlshelper/tlshelper.go:38]:
```suggestion
// TLSHelper struct
```

@nitu-s-gupta Same trivial suggestion from me. 😄

**Review by MoonkiHong — CHANGES_REQUESTED** [review]
@nitu-s-gupta Almost your PR looks good, but PTAL at my several comments. 😁

File: internal/restinterface/resthelper/client/tlshelper/tlshelper.go (hunk 5 above)
Comment (nitu-s-gupta) [reply internal/restinterface/resthelper/client/tlshelper/tlshelper.go]: Done

File: internal/restinterface/resthelper/client/httphelper/httphelper.go (hunk 4 above)
Comment (nitu-s-gupta) [reply internal/restinterface/resthelper/client/httphelper/httphelper.go]: DOne

File: internal/controller/servicemgr/types.go (hunk 3 above)
Comment (nitu-s-gupta) [reply internal/controller/servicemgr/types.go]: Done

File: internal/common/resourceutil/cpu/cpu.go (hunk 2 above)
Comment (nitu-s-gupta) [reply internal/common/resourceutil/cpu/cpu.go]: Done

**Review by MoonkiHong — APPROVED** [review]
+1 😄

**Pull request merged**